ert3/engine/_run.py

```diff
@@ -143,69 +156,47 @@ def _load_sensitivity_parameters(
         record_source = input_record.source.split(_SOURCE_SEPARATOR)
         if record_source[0] == "stochastic":
             assert len(record_source) == 2
-            parameter_group_name = record_source[1]
-            sensitivity_parameters[record_name] = all_distributions[
-                parameter_group_name
-            ]
+            parameter_name = record_source[1]
```

Comment:
Then actually `group_name` is a better shortening, as otherwise one could be fooled into thinking it is the name of a single variable.